docs/csharp/language-reference/keywords/init.md

```diff
@@ -28,6 +28,9 @@ var john = new Person_InitExample
 
 john.YearOfBirth = 1926; //Not allowed, as its value can only be set once in the constructor
 ```
+Do note, that this behavior does not change when nullablity is used. `init` properties do not force the values to be initialized. What `init` does is that, if the property is to be initialized, it must be done at construction time in an object initializer (or in the constructor if you have one). So the behavior is the same for this example:
```

Comment:
Two suggestions here:  First, add a blank line for markdown lint. Second, I made some clarifying comments on the text:

```suggestion

An `init` accessor doesn't force callers to set the property. Instead, it allows an object initializer to set the initial value while prohibiting later modification. You can add the `required` modifier to force callers to set a property. The following example shows the same behavior:
```

Reply:
Agreed, my wording is a bit... direct. I like your wording alot better.